environment/src/main/java/com/sequenceiq/environment/environment/service/PollerCollection.java

```diff
@@ -0,0 +1,218 @@
+package com.sequenceiq.environment.environment.service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
remove this please

Reply:
javax.validation.constraints.NotNull should be used